app/parse.py

```diff
@@ -8,8 +14,38 @@ class Quote:
     tags: list[str]
 
 
+def get_single_quote(quote) -> Quote:
+    return Quote(
+        text=quote.find("span", itemprop="text").text,
+        author=quote.find("small").text,
+        tags=[tag.text for tag in quote.find_all("a", class_="tag")],
+    )
+
+
+def get_quotes() -> [Quote]:
```

Comment:
The return type hint for `get_quotes` should be `list[Quote]` instead of `[Quote]` to correctly indicate a list of `Quote` objects.